docker-compose.yml

```diff
@@ -2,20 +2,13 @@ version: "3"
 
 services:
 
-  dynamo:
-    image: amazon/dynamodb-local
-    ports:
-    - "8000:8000"
-
   test:
     build: .
-    volumes:
-    - ./:/certmagic-storage-dynamodb
-    environment:
-      AWS_ENDPOINT: dynamo:8000
-      AWS_DEFAULT_REGION: us-east-1
-      AWS_ACCESS_KEY_ID: abc123
-      AWS_SECRET_ACCESS_KEY: abc123
-    depends_on:
-      - dynamo
     command: make
+    environment:
+      SKYDB_ENDPOINT: localhost:9980
+      SKYDB_SEC_KEY: R2ZwfGghZHm75i0b5Wv12P1jm6ULtIthcAszSZuH5S482JTaSwfi8m+AlvhqCmwFYKyi3Ev53a7MbJ7yUGxiBw==
```

Comment:
Did we mean to expose the secret key?

Reply:
This is only used for testing. The real secret/public keys are taken from environment variables that we're going to define per server (they should still be the same though).